src/kOS/Screen/TermWindow.cs

```diff
@@ -178,6 +181,11 @@ private void LoadFontArray()
             }
         }
         
+        public kOS.Safe.Sound.ISoundMaker GetSoundMaker()
```

Comment:
My intent was to expose `GetSoundMaker()` to `kOS.Safe` so it could be called from kOS.Safe without breaking (it would be just a dummy do-nothing if Unity's not there, but it wouldn't break the compile).  But the way `SharedObjects` vs `SafeSharedObects` works, I can't do that interface abstraction at the level of `SharedObjects`, I have to do it on one of the items *in* `SharedObjects`, and `ITermWindow` made more sense than any of the others did.

But in the end I didn't end up needing to expose anything about it after all, so it didn't end up in `ITermWindow`.  But I prefer it the way it is now because it leaves open the chance of putting it there later if we need to.